src/i18n/strings/en_EN.json

```diff
@@ -868,7 +868,7 @@
         "hidebold": "Hide notification dot (only display counters badges)",
         "intentional_mentions": "Enable intentional mentions",
         "ask_to_join": "Enable ask to join",
-        "new_room_decoration_ui": "Under active development, new room header & details interface"
+        "new_room_decoration_ui": "New room header & details interface"
```

Comment:
Isn't this now unused?

Reply:
Still used here, https://github.com/matrix-org/matrix-react-sdk/pull/11507/files#diff-56fe49f3c6514890fd8404b8bfc9e1351f29839e8dfd98931872e5acab84e2f7L550